actstream/feeds.py

```diff
@@ -242,6 +242,11 @@ def get_object(self, request, content_type_id, object_id):
     def get_stream(self):
         return any_stream
 
+class StreamKwargsMixin(object):
```

Comment:
explicit inheritance from the object is not needed.

Reply:
Yeah there is no need to do that. But I followed the pattern of other classes in ``feeds.py`` like ``AbstractActivityStream``, ``ModelActivityMixin``, etc.

Also I will add tests for ``StreamKwargsMixin``